releasing_guide.md

```diff
@@ -18,9 +18,13 @@ To publish a new LumiSpy release do the following steps:
 - Create a tag e.g. `git tag -a v0.1.1 -m "LumiSpy version 0.1.1"`
 - Push tag to user fork for a test run `git push origin v0.1.1`. Will run the release
   workflow without uploading to PyPi
-- Push tag to LumiSpy repository to trigger release `git push upstream v0.1.1`
-  (this triggers the GitHub action to create the sdist and wheel and upload to
-  PyPi automatically). :warning: this is a point of no return :warning:
+- Push tag to LumiSpy repository to trigger release `git push upstream v0.1.1`. 
+  :warning: this is a point of no return :warning:
+  - triggers the GitHub action to create the sdist and wheel and upload them to
+  PyPi automatically
+  - creates a Github Release
+  - creates a zenodo record and corresponding DOI (triggered by a new release)
+  _updates the current version on readthedocs to this release [we would need to set it to stable instead of latest]
```

Comment:
```suggestion
  - updates the current version on readthedocs to this release [we would need to set it to stable instead of latest]
```
This should be done automatically (to check), but it is not clear how `readthedocs.org` does it, i.e. update the `stable` version and it needs a specific setting. It is most likely trigger by a tag push rather the github release creation.

Reply:
Currently, readthedocs is set to the `latest` version, thus the development version of the user guide. That was to get a first version running while it was not released/work in progress. However, as most topics are now covered, we should change it to `stable` sometime so that the docs correspond to the released version. Then this point would be valid and the commenting out can be removed.

Reply:
Oh, I thought this was a typo! 😅